specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/examples/SessionHostConfigurations_Create.json

```diff
@@ -42,6 +43,9 @@
   },
   "responses": {
     "200": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/providers/Microsoft.DesktopVirtualization/locations/westus/operationStatuses/ed5d502c-acaa-42ec-ad61-0d8488a9fd1d"
+      },
       "body": {
         "name": "default",
         "id": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourceGroups/resourceGroup1/providers/Microsoft.DesktopVirtualization/hostPools/hostPool1/updateStatuses/current",
```

Comment:
updateStatuses here should be updateDetails right?

Reply:
this is sessionhost configuration not hostpool update